openid-4-verifiable-presentations-1_0.md

```diff
@@ -705,6 +715,89 @@ The rules for selecting claims to send are defined in (#selecting_claims).
 
 Note that multiple Credential Queries in a request MAY request a presentation of the same Credential.
 
+### Trusted Authorities Query {#dcql_trusted_authorities}
+
+A Trusted Authorities Query is an object representing information that helps to identify an issuer
+or the trust framework that an issuer is belonging to. A Credential is identified as a match
+to a Trusted Authorities Query if it to matches with one of the provided values in one of the provided
+types. How exactly the matching works is defined for the different types below
+
+Note that explicit issuer matching can also be achieved using claim value matching for certain
+Credential formats (e.g., value matching the `iss` claim in an SD-JWT). The methods provided in
+`trusted_authorities` are tailored towards matching trust frameworks and more direct issuer matching
+variants that require more complex matching logic (e.g., certificate thumbprints).
```

Comment:
Certificate thumbprints are not part of this PR anymore. Would aki suffice as an example here? Or wouldn't that be called direct issuer matching?

Reply:
Changed the example to AKI